### ISSUE TITLE
Alternative to sshfs

### ISSUE THREAD

**Issue by JakubJecminek** [lima-vm/lima#487]
Hello, 
after upgrade to Mac Os X Monterey and brew upgrade I have problem with lima that is caused by sshfs being disabled. This is because sshfs depends on macfuse which is no longer under open source license: 

- https://github.com/Homebrew/discussions/discussions/1760#discussioncomment-945235
- https://github.com/osxfuse/osxfuse/issues/616

When you try to (re)install sshfs, homebrew will give you following error:
```
brew install sshfs
Error: sshfs has been disabled because it requires closed-source macFUSE!
```

Is there alternative to sshfs that would support same functionality and that is compatible with lima?

**Comment by afbjorklund** [comment]
Support for qemu virtfs is available (uses 9p to mount), but yet not merged into qemu master (for Mac).

If qemu is patched, or once the patches land upstream, it would be possible to mount using that instead...

Similar to podman:
* https://github.com/containers/podman/pull/11454

But see also:
* #188

**Comment by afbjorklund** [comment]
@JakubJecminek : pardon my ignorance, but I thought that `sshfs` was only needed on the Linux VM ?

And that the host would use `sftp` to serve the files, at least that is how it is implemented on Linux afaik

**Comment by AkihiroSuda** [comment]
sshfs is solely used inside the guest, not on the host.
You do NOT need to run sshfs on the host.

**Comment by JakubJecminek** [comment]
Ohh ok thank you. Sorry for the wrong question :) And thank you very much for this great project.

**Comment by dmahurin** [comment]
Perhaps this will work for some still needing sshfs.

It is a shell script that provides a basic sshfs command line interface, but mounts files with ssh tunneled webdav and mount_webdav.

https://github.com/dmahurin/sshfs-dav/
